livefs_edit/actions.py

```diff
@@ -333,7 +333,7 @@ def add_autoinstall_config(ctxt, autoinstall_config):
 
 
 @register_action()
-def resign_pool(ctxt):
+def resign_pool(ctxt, dist=None):
```

Comment:
The reason I wanted to handle the dist=None case is that resign-pool is available as an action all by itself. But that's trivial, I can make that change :-)